src/main/assisted-installer-controller/assisted_installer_main.go

```diff
@@ -28,20 +28,25 @@ func main() {
 
 	kc, err := k8s_client.NewK8SClient("", logger)
 	if err != nil {
-		log.Fatalf("Failed to create k8 client %e", err)
+		log.Fatalf("Failed to create k8 client %v", err)
```

Comment:
lets move it to log.WithError

Reply:
log doesn't have 'WithError', the assisted-service uses a different logging lib